package.json

```diff
@@ -50,7 +51,7 @@
     "_lint:styles:fix": "npm run --silent _lint:styles -- --fix",
     "_lint:styles:changed": "node ./scripts/get-changed-files.js --command \"stylelint\" --filter \"**/*.{css,scss,sass}\"",
     "_lint:styles:changed:fix": "node ./scripts/get-changed-files.js --command \"stylelint --fix\" --filter \"**/*.{css,scss,sass}\"",
-    "_start:storybook": "wait-on ./documentation.json ./dist/clr-ui/clr-ui.css && start-storybook -p 6006"
+    "_start:storybook": "wait-on ./documentation.json ./dist/clr-ui/clr-ui.css && STORYBOOK_ANGULAR_PROJECT=clr-angular start-storybook -p 6006"
```

Comment:
This change isn't needed since we are skipping the storybook build for Angular v14 and v15.